docs/upgrade-notes/upgrade-rds.md

```diff
@@ -0,0 +1,246 @@
+
+Then you can re-deploy your data-persistence module as usual:
+
+```bash
+terraform apply
+```
+
+### 3. Deploy and run data-migration1
+
+You will need to create a `data-migration1-tf` directory. See the configuration in our template-deploy repo for reference: <https://github.com/nasa/cumulus-template-deploy/tree/master/data-migration1-tf>
+
+Navigate to the directory `data-migration1-tf` and copy the following `.example` files:
+
+```shell
+cp terraform.tf.example terraform.tf
+cp terraform.tfvars.example terraform.tfvars
+```
+
+In `terraform.tf`, configure your remote state settings by replacing the appropriate value for `PREFIX`.
+
+In `terraform.tfvars` replace the appropriate values for the following variables:
+
+- `prefix`
+- `data_persistence_remote_state_config`
+- `permissions_boundary_arn`
+- `lambda_subnet_ids`
+- `vpc_id`
+- `provider_kms_key_id`
+
+These values should match the values used for your Cumulus Core deployment.
+
+After replacing those values, run `terraform init`.
+The output should resemble the following:
+
+```shell
+Initializing modules...
+
+Initializing the backend...
+
+Initializing provider plugins...
+- Using previously-installed hashicorp/aws v3.34.0
+- terraform.io/builtin/terraform is built in to Terraform
+
+Terraform has been successfully initialized!
+```
+
+Run  `terraform apply` to deploy `data-migration1` and type `yes` when prompted to create those resources.
+On success, you will see output like:
+
+```shell
+Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
+```
+
+Once the deployment is complete, you can use the AWS Console or CLI to invoke the Lambda and start the data migration:
+
+```bash
+aws lambda invoke --function-name $PREFIX-data-migration1 $OUTFILE
+```
+
+where
+
+- `OUTFILE` is the filepath to store the output from the Lambda.
+- `PREFIX` is the `prefix` value used to deploy data-migration1-tf
+
+### 4. Deploy Cumulus module
+
+The following variables were added to the Cumulus module
+
+- `rds_security_group`
+- `rds_user_access_secret_arn`
+- `rds_connection_heartbeat`
+
+For reference on how to set these values, see our template-deploy repo: <https://github.com/nasa/cumulus-template-deploy/tree/master/cumulus-tf>
+
+Instructions on deploying the Cumulus module can be found [here](./../deployment/README.md).
+
+The `cumulus` module will create resources including the following relevant resources for the data migration:
+
+- `${PREFIX}-data-migration2` Lambda
+- `${PREFIX}-postgres-migration-async-operation` Lambda
+
+### 5. Run the second data migration
+
+Now that Cumulus module is deployed, we can use some newly created resources to migrate granule, execution, and PDR data from DynamoDB to our PostgreSQL database.
+
+This second data migration process can be run by invoking the provided `${PREFIX}-postgres-migration-async-operation` Lambda included in the Cumulus module deployment.
```

Comment:
We provide `aws lambda invoke` command samples for every other place we invoke stuff, but it's missing here. Suggest adding one.

Reply:
I guess it's on line 128, but I was confused by the incorrect function name.